linker/Tests/TestCasesRunner/AssemblyChecker.cs

```diff
@@ -371,6 +372,30 @@ protected virtual IEnumerable<string> FilterLinkedSecurityAttributes (ISecurityD
 				.Select (attr => attr.AttributeType.ToString ());
 		}
 
+		void VerifyDelegateBackingFields (TypeDefinition src, TypeDefinition linked)
+		{
+			// This behavior is unique to mcs.  If mcs was not used to compile the test case we can return
+			if (!UsingMcsCompiler)
```

Comment:
I don't think this is necessary

Reply:
It's for our UnityLinker test suites.  We always use mcs, even on Windows.

Reply:
Just remove the check and it's still going to work for you

Reply:
Ah, I see what you mean now.  Because I'm looping over the src type's fields, the fields won't be there, so it will never try to verify they are there.

Reply:
Done